.github/workflows/ci-bundler.yml

```diff
@@ -0,0 +1,36 @@
+name: CI Bundler
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - Dockerfile
+      - common
+      - bundler
```

Comment:
I wanted to highlight to anyone concerned with this change that _all_ workflows will run when either `common/`  or `Dockerfile` change.

Reply:
Assuming I can get the pattern right!